src/firefly/html/slate.html

```diff
@@ -17,14 +17,14 @@
                     {label:'Images', action:'ImageSelectDropDownSlateCmd'},
                     {label:'Catalogs', action:'IrsaCatalogDropDown'},
                     {label:'Charts', action:'ChartSelectDropDownCmd'},
-                    ],
-                options : {
-                    MenuItemKeys: {maskOverlay:true},
-                    catalogSpacialOp: 'polygonWhenPlotExist',
-                    disableDefaultDropDown: true,
-                    hips : {useForCoverage: true, useForImageSearch: true},
-                    charts: {multitrace: true}
-                }
+                ],
+            },
+            options : {
+                MenuItemKeys: {maskOverlay:true},
+                catalogSpacialOp: 'polygonWhenPlotExist',
+                disableDefaultDropDown: true,
+                hips : {useForCoverage: true, useForImageSearch: true},
```

Comment:
hips should be removed since it will pickup the default. I think hips is defaulted now.